stdlib/src/collections/list.mojo

```diff
@@ -20,11 +20,13 @@ from collections import List
 """
 
 
-from builtin.value import StringableCollectionElement
+from builtin.value import (
```

Comment:
**Suggestion** You can remove this entirely.  For public entities in the `builtin` module, you don't need to import anything.  They will automatically be imported into every Mojo program.